src/main/java/me/roinujnosde/titansbattle/games/Game.java

```diff
@@ -59,6 +59,9 @@ public Warrior findKiller() {
     @Override
     public void start() {
         super.start();
+        if (plugin.getConfig().getBoolean("webhook")) {
```

Comment:
This is already checked in TitansBattle#sendDiscordMessage. Can be removed here.